app/routes.py

```diff
@@ -1,2 +1,54 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, moons, description):
+        self.id = id
+        self.name = name
+        self.moons = moons
+        self.description = description 
+
+
+planets = [
+    Planet(1, "Mercury", 0, "The first planet."),
+    Planet(2, "Venus", 0, "The second planet."),
+    Planet(3, "Earth", 1, "The third planet.")
+
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+
+@planets_bp.route("", methods=["GET"])
+def handle_planets():
```

Comment:
Conventionally you would see the naming of the function be similar to the CRUD operation you are performing. Like `get_planets` or `get_all_planets`